http/js/script.js

```diff
@@ -80,19 +80,12 @@ function gaussian(x, a, b, c) {
 }
 
 $(document).ready(function() {
-    $(".footnote a").tipsy({gravity: "w"});
-//    $(".footnote a").on("click", function(e) {
-//        if ($(".tipsy").length === 0) {
-//            $(this).tipsy("show");
-//        }
-//        else {
-//            $(this).tipsy("hide");
-//        }
-//    });
+    // Add tipsy for the footnote.
+    $('.footnote').tipsy({title: 'data-footnote', gravity: 'w', fade: true});
```

Comment:
Out of interest, what was the rationale behind changing from `title` to `data-footnote`? The `title` attribute is suited to this purpose, of displaying a tooltip on an element. It also works if JS is disabled (albeit without the fancy Tipsy tooltip)